langchain4j-milvus/src/test/java/dev/langchain4j/store/embedding/milvus/MilvusEmbeddingStoreIT.java

```diff
@@ -65,6 +69,10 @@ void should_not_retrieve_embeddings_when_searching() {
                 .consistencyLevel(STRONG)
                 .dimension(384)
                 .retrieveEmbeddingsOnSearch(false)
+                .idFieldName("id_field")
```

Comment:
Why is this change required? I can understand setting these fields explicitly in one of the tests, but this should not be required for all of them

Reply:
The embeddingStore define in class level share the some instance with the one defined in method level. So their field name must be same.